### PR TITLE
Fix #52, Apply consistent Event ID names to common events

### LINKED ISSUE #52: Inconsistent Event ID naming
**Checklist**
* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Copy of https://github.com/nasa/cFE/issues/2175
After finding that there were 9 different Event IDs to indicate the same thing (Invalid Message ID) in https://github.com/nasa/CF/issues/262, I scrubbed the other common commands (e.g. Task Initialisation [INIT], NOOP, Reset Counters etc.) and found the same issue there - almost every component/app had their own variation of the Event ID name for the exact same event.

**Expected behavior**
Apply consistent Event ID names to the events which are common to all/most components and apps.

**Code snips**
Invalid Message ID:
`CFE_EVS_ERR_MSGID_EID`
`CFE_SB_BAD_MSGID_EID`
`CFE_TIME_ID_ERR_EID`
`CS_MID_ERR_EID`
`TO_LAB_MSGID_ERR_EID`
`SAMPLE_APP_INVALID_MSGID_ERR_EID`
`BP_INVALID_MID_ERR_EID`
`SCH_MD_ERR_EID`
`CI_LAB_COMMAND_ERR_EID`

Initialization:
`CFE_TIME_INIT_EID`
`CFE_TBL_INIT_INF_EID`
`CFE_EVS_STARTUP_EID`
`CF_EID_INF_INIT`
`BP_INIT_APP_INFO_EID`
`SCH_INITSTATS_INF_EID`
`CI_LAB_STARTUP_INF_EID`

NOOP:
`CFE_TIME_NOOP_EID`
`CFE_TBL_NOOP_INF_EID`
`CFE_SB_CMD0_RCVD_EID`
`CF_EID_INF_CMD_NOOP`
`FM_NOOP_CMD_EID`
`CI_LAB_COMMANDNOP_INF_EID`

Reset Counters:
`CFE_TIME_RESET_EID`
`CFE_TBL_RESET_INF_EID`
`CFE_EVS_RSTCNT_EID`
`CFE_SB_CMD1_RCVD_EID`
`CF_EID_INF_CMD_RESET`
`SC_RESET_DEB_EID`
`HS_RESET_DBG_EID`
`FM_RESET_CMD_EID`
`HK_RESET_CNTRS_CMD_EID`
`MD_RESET_CNTRS_DBG_EID`
`CI_LAB_COMMANDRST_INF_EID`

etc.

**Reporter Info**
Avi Weiss @thnkslprpt

### PR DESCRIPTION
**Pull request by thnkslprpt**
**Checklist**
* [x] I reviewed the [Contributing Guide](https://github.com/nasa/osal/blob/main/CONTRIBUTING.md).
* [x] I signed and emailed the appropriate [Contributor License Agreement](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to GSFC-SoftwareRelease@mail.nasa.gov and copied cfs-program@lists.nasa.gov.

**Describe the contribution**
- Fixes #52
  - Consistent event IDs have been applied to the inconsistent cases to align them with a common Event ID naming convention.

**Testing performed**
Only GitHub CI actions.

**Expected behavior changes**
No impact on code behavior (no logic changes).
Consistent Event ID names for the events which are common to all/most cFS components and apps will improve consistency and ease make code review/debugging easier.

**Contributor Info**
Avi Weiss @thnkslprpt

### CODE REVIEW

File: fsw/src/cs_events.h [hunk 1]
```diff
@@ -60,7 +60,7 @@
  *
  *  This event message is issued when a reset counters command has been received.
  */
-#define CS_RESET_DBG_EID 3
+#define CS_RESET_INF_EID 3
```
Comment (chillfig) [inline fsw/src/cs_events.h:63]: Why does this event name change from using DBG to INF? 

Line 57 may need an update to indicate this event is informational type.

**Review by chillfig — CHANGES_REQUESTED** [review]
`CS_RESET_INF_EID` has header comment in its definition that suggest it is Debug type. May just need an update on the header comment to change from Type: DEBUG to Type: INFORMATION

File: fsw/src/cs_events.h (hunk 1 above)
Comment (thnkslprpt) [reply fsw/src/cs_events.h:63]: Should be updated now Justin.

**Review by chillfig — APPROVED** [review]
Looks good!

File: fsw/src/cs_table_processing.c [hunk 2]
```diff
@@ -109,6 +109,7 @@ int32 CS_ValidateEepromChecksumDefinitionTable(void *TblPtr)
                                   "Eeprom Table Validate: Illegal State Field (0x%04X) found in Entry ID %d",
                                   (unsigned short)StateField, (int)OuterLoop);
                 Result = CS_TABLE_ERROR;
+                CS_DisableAllCSCmd()
```
Comment (chillfig) [inline fsw/src/cs_table_processing.c:112]:
```suggestion
                CS_DisableAllCSCmd();
```

**Comment by thnkslprpt** [comment]
> Looks good!

Cheers mate.

**Review by chillfig — CHANGES_REQUESTED** [review]
I missed this. fsw/src/cs_table_processing.c/line 112 needs a semi-colon.

File: fsw/src/cs_table_processing.c (hunk 2 above)
Comment (thnkslprpt) [reply fsw/src/cs_table_processing.c]: That wasn't supposed to be in there (not sure how it got pasted in...).
This PR is only for Event ID updates.
Have removed the change to cs_table_processing.c

**Review by chillfig — APPROVED** [review]

**Comment by chillfig** [comment]
CCB:2023.03.09: Putting on hold until naming conventions for event, etc are merged.

**Comment by jphickey** [comment]
It looks like in the recent rebase, some updates got merged incorrectly, causing the workflow failure

**Comment by jphickey** [comment]
Rebased to latest mainline and manually fixed up some merge overlaps re: payload member.  Workflows now pass and this should be good to go.

**Comment by thnkslprpt** [comment]
Thanks Joe.

**Pull request merged**